### PR TITLE
Add a task to easily release a new EigenPy version

### PR DESCRIPTION
**Pull request by jorisv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,6 +207,24 @@ pixi run test
 The project will be built in the `build` directory.
 You can run `pixi shell` and build the project with `cmake` and `ninja` manually.
 
+## Release with Pixi
```
Comment (jcarpent) [inline README.md:210]: I would suggest moving specialized topics to a dedicated readme section to clear the current one.
We can put many things in release.md for instance.

**Review by jcarpent — COMMENTED** [review]
Otherwise, the PR looks very good to me.

File: README.md (hunk 1 above)
Comment (jorisv) [reply README.md:210]: Ok, did using the development sub directory is fine to you ?

We can have the following file inside
- development/release.md
- development/scripts/pixi/activate.*
- development/contributing.md

Comment (jcarpent) [reply README.md:210]: Perfect

**Review by olivier-roussel — APPROVED** [review]

**Review by jcarpent — APPROVED** [review]

**Pull request merged**